ignite/metrics/ssim.py

```diff
@@ -0,0 +1,171 @@
+from typing import Callable, Sequence
+
+import torch
+import torch.nn.functional as F
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = ["SSIM"]
+
+
+class SSIM(Metric):
+    """
+    Computes Structual Similarity Index Measure
+
+    Args:
+        kernel_size: Size of the gaussian kernel. Default: (11, 11)
```

Comment:
@ydcjeff please, add types for those args like that
```suggestion
        kernel_size (list or tuple of int): Size of the gaussian kernel. Default: (11, 11)
```